peekaboo/ruleset/rules.py

```diff
@@ -224,22 +225,42 @@ def evaluate(self, sample):
                            True)
 
 
-class OfficeMacroWithAutoActionRule(Rule):
+class OfficeMacroWithSuspiciousKeyword(Rule):
     """ A rule checking the sample for Office macros. """
-    rule_name = 'office_macro_with_auto_action'
+    rule_name = 'office_macro_with_suspicious_keyword'
+
+    def get_config(self):
+        # get list of keywords from config file
+        suspicious_keyword_list = self.get_config_value(
+            'keyword', [])
+        # convert into list of regexes
```

Comment:
Huh, there must be a way to avoid this duplication by allowing to pass flags like re.IGNORECASE to `PeekabooConfigParser.getrelist`. We could encode it in an option type self.config.IRELIST, the I meaning IGNORECASE. Or we could actually have classes derived from list called ReList and IReList and so on providing the respective compile function. In both cases we'd need a small `getirelist()` wrapper for `getrelist()` or an additional parameter to `get_by_type()` to pass type-specific flags such as IGNORECASE.

Reply:
Solved quite nicely for now with IRELIST config type.